### PR TITLE
Removed the generated code in flag files which is not used

### PR DESCRIPTION
**Pull request by pmahindrakar-oss**
Signed-off-by: Prafulla Mahindrakar <prafulla.mahindrakar@gmail.com>

# TL;DR

The generated code function of `elemValueOrNil` and `mustMarshalJSON` are not used in flytectl 
And also the corresponding generated tests for these files don't cover these functions and ultimately reducing the coverage .

Since we have added many flags for the commands , this is overall reducing the UT coverage of the entire repo.

Need to understand more details on when this code will be utilized and if there is flag to not generate it.
This particular line is generating this code 

https://github.com/flyteorg/flytestdlib/blob/99f4b2005d747fb9b4807adfa3e7c27375a5fd35/cli/pflags/api/generator.go#L185

Increases the code coverage by  +5.58%

## Type
 - [ ] Bug Fix
 - [ ] Feature
 - [ ] Plugin

## Are all requirements met?

 - [ ] Code completed
 - [ ] Smoke tested
 - [ ] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description
 _How did you fix the bug, make the feature etc. Link to any design docs etc_

## Tracking Issue
https://github.com/lyft/flyte/issues/<number>

## Follow-up issue
_NA_
OR
_https://github.com/lyft/flyte/issues/<number>_

### CODE REVIEW

**Comment by pmahindrakar-oss** [comment]
@EngHabu please have a look when you get a chance.

**Comment by evalsocket** [comment]
same issue. https://github.com/flyteorg/flytectl/pull/99 my pr has some changes in sandbox package and sandbox package coverage is around 80% that is more than our requirement but because of pflag, My overall coverage is 59% and my check failed
